src/components/UserData.spec.js

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react'
+import UserData from './UserData'
+
+describe('UserData', () => {
+  it('renders a List with personal data and a skills', () => {
+    render(
+      <UserData
+        personalUserData={[
+          'Peter Parker',
+          '007 007 007',
+          'peter@spider.web',
+          'Kitchen',
+        ]}
+        skills={['SKILL1', 'SKILL2', 'SKILL3', 'SKILL4', 'SKILL5', 'SKILL6']}
```

Comment:
Again, I would only give one prop `userData` and have it as an object, like I wrote before.